### PR TITLE
ILIAS6: UI Components, fix of responsive Logo Issue, see: #32016

### PR DESCRIPTION
**Pull request by thibsy**
Currently, the responsive Header Logo is not displayed on small screen sizes, the standard one is. See: https://mantis.ilias.de/view.php?id=32016

This PR is based on https://github.com/ILIAS-eLearning/ILIAS/pull/4029 introduced by @Amstutz with ILIAS 6 compatibility.

### CODE REVIEW

**Review by Amstutz — APPROVED** [review]
See comment in https://github.com/ILIAS-eLearning/ILIAS/pull/4129. @thibsy shortly talked about the introduced strict types. We agree that this would be better for only trunk normally, however since they have been manually tested by @thibsy I am fine with them in this case. @klees do you want to check the cli part?

File: setup/cli.php [hunk 1]
```diff
@@ -67,9 +67,13 @@ function build_container_for_setup(string $executed_in_directory)
             $c["command.install"],
             $c["command.update"],
             $c["command.build-artifacts"],
-            $c["command.reload-control-structure"]
+            $c["command.reload-control-structure"],
+            $c["command.replace-logo"]
```
Comment (chfsx) [inline setup/cli.php]: @thibsy not sure if this need an own command like this, maybe only a Objective would be sufficient?

Comment (thibsy) [reply setup/cli.php]: Would be sufficient, I implemented a command to simplify the composer.json entry. However, something like this should also be possible: `php -r "(new ReplaceResponsiveLogoObjective())->execute(new ArrayEnvironment([]))"`.

**Comment by thibsy** [comment]
As per @chfsx's suggestion I moved the implementation as a command to an objective that has been added to the ILIAS update process. This makes more sense because the replacement doesn't need to take place on every composer dump.

**Review by klees — APPROVED** [review]
I didn't check the algorithm of the `Objective` in detail, but the form and inclusion looks just fine.

Remind yourself that you can remove it with ILIAS 9 because of our [new update policy](https://github.com/ILIAS-eLearning/ILIAS/blob/trunk/docs/development/supported-versions.md#supported-versions).

**Comment by thibsy** [comment]
@klees yes, I already removed the objective for the ILIAS 8 PR (#4131). 

I'll merge the PRs then =).

**Pull request merged**